account/src/use-case/createUserAccount.js

```diff
@@ -0,0 +1,22 @@
+let contas = []
+const data = new Date();
+let dataAtualFormatada = data.toISOString().substring(0, 10);
```

Comment:
Recomendo por essa data dentro da função, isso por que essa data não vai mudar depois que este arquivo for lido/executado. A não ser que você lembre de mudar.